### PR TITLE
Invite Talent UI

### PR DESCRIPTION
**Pull request by GabisPapin**
-created a page with the freelancer's information;
-created a popup message to offer a job;
-created a popup message in case of the job owner don't have job post.

### CODE REVIEW

**Comment by GabisPapin** [comment]
invite-talent-page
popup-job-post
popup-invitation

File: src/components/inviteTalent/inviteTalent.styles.ts [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+	align-content: center;
+	display: flex;
+	flex-flow: column wrap;
+	font-weight: bold;
+	margin-top: 10px;
+	padding: 10px 5px;
+	width: 100%;
+`;
+
+export const TextArea = styled.textarea`
```
Comment (lyudad) [inline src/components/inviteTalent/inviteTalent.styles.ts]: why you create your if antd design have componets, and project already used text areas, its very bad practice create everything again and again

File: src/components/inviteTalent/inviteTalent.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+						<H5>{el.price}</H5>
+					</Div1>
+					<Div3>
+						<P>{el.description}</P>
+					</Div3>
+				</>
+			))}
+			<Invite type="button" onClick={() => setOpen(true)}>{`${t('InvitePage.button')}`}</Invite>
+			{post ? (
+				<Popup open={open} closeOnDocumentClick onClose={() => setOpen(false)}>
```
Comment (lyudad) [inline src/components/inviteTalent/inviteTalent.tsx]: why it in the came component, it make component big, split

File: src/components/inviteTalent/inviteTalent.styles.ts (hunk 1 above)
Comment (GabisPapin) [reply src/components/inviteTalent/inviteTalent.styles.ts]: I fixed it.

File: src/components/inviteTalent/inviteTalent.tsx (hunk 2 above)
Comment (GabisPapin) [reply src/components/inviteTalent/inviteTalent.tsx]: I created a popup component.

File: src/components/inviteTalent/component/Invitepopup.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+const TEXTAREA_ROWS_MAX = 16;
+const TEXTAREA_ROWS_MIN = 8;
+const BORDER_RADIUS = 6;
+
+const InvitePopup = () => {
+	const { t } = useTranslation();
+	const navigate = useNavigate();
+	const [isDisebled, setIsDisebled] = useState(false);
```
Comment (lyudad) [inline src/components/inviteTalent/component/Invitepopup.tsx]: types

File: src/components/inviteTalent/component/Invitepopup.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+	const [open, setOpen] = useState(false);
```
Comment (lyudad) [inline src/components/inviteTalent/component/Invitepopup.tsx]: types

**Pull request merged**